packages/remix/README.md

```diff
@@ -6,4 +6,128 @@
 
 # Official Sentry SDK for Remix
 
-This SDK is work in progress, and should not be used before officially released.
+[![npm version](https://img.shields.io/npm/v/@sentry/nextjs.svg)](https://www.npmjs.com/package/@sentry/remix)
+[![npm dm](https://img.shields.io/npm/dm/@sentry/nextjs.svg)](https://www.npmjs.com/package/@sentry/remix)
+[![npm dt](https://img.shields.io/npm/dt/@sentry/nextjs.svg)](https://www.npmjs.com/package/@sentry/remix)
+[![typedoc](https://img.shields.io/badge/docs-typedoc-blue.svg)](http://getsentry.github.io/sentry-javascript/)
+
+This SDK is considering experimental and in an alpha state. It may experience breaking changes. Please reach out on [GitHub](https://github.com/getsentry/sentry-javascript/issues/new/choose) if you have any feedback/concerns.
+## General
+
+This package is a wrapper around `@sentry/node` for the server and `@sentry/react` for the client, with added functionality related to Remix.
+
+To use this SDK, initialize Sentry in your Remix entry points for both the client and server.
+
+```ts
+// entry.client.tsx
+
+import { useLocation, useMatches } from "@remix-run/react";
+import * as Sentry from "@sentry/remix";
+import { useEffect } from "react";
+
+Sentry.init({
+  dsn: "__DSN__",
+  tracesSampleRate: 1,
+  integrations: [
+    new Sentry.BrowserTracing({
+        routingInstrumentation: Sentry.remixRouterInstrumentation(
```

Comment:
nit: Why don't we set these properties for the user inside of `remixRouterInstrumentation` instead of asking them to do this?

Reply:
Because dependency injection means that we don’t have to deal with importing the functions ourselves, which means we minimize versioning conflicts.

also init is usually only done once, so it’s not like folks have to come back and touch this code again, so just letting them pass in the imports is fine.

we’ve done this for the rest of our routing instrumentation and never gotten negative feedback.